### PR TITLE
Add info about orb categories 

### PR DESCRIPTION
**Pull request by lokst**
# Description
Add information about how to categorize an orb.

# Reasons
https://circleci.atlassian.net/browse/CIRCLE-29338

### CODE REVIEW

File: jekyll/_cci2/orb-author.md [hunk 1]
```diff
@@ -278,3 +278,50 @@ Read our full [Orb Testing Methodologies]({{site.baseurl}}/2.0/testing-orbs/) do
 With the orb development kit, a fully automated CI and CD pipeline is automatically configured within `.circleci/config.yml`. This configuration makes it simple to automatically deploy semantically versioned releases of your orbs.
 
 For more information, see the [Orb Publishing Process]({{site.baseurl}}/2.0/creating-orbs/) guide.
+
+### Categorizing Your Orb
+
+You can categorize your orb for better discoverability in the [Orb Registry](https://circleci.com/developer/orbs/). Categorized orbs are searchable by category in the [Orb Registry](https://circleci.com/developer/orbs/). (CircleCI may also from time to time create or edit orb categorizations to improve orb discoverability).
```
Comment (rosieyohannan) [inline jekyll/_cci2/orb-author.md:284]:
```suggestion
You can categorize your orb for better discoverability in the [Orb Registry](https://circleci.com/developer/orbs/). Categorized orbs are searchable by category in the [Orb Registry](https://circleci.com/developer/orbs/). CircleCI may, from time to time, create or edit orb categorizations to improve orb discoverability.
```

File: jekyll/_cci2/orb-author.md [hunk 2]
```diff
@@ -278,3 +278,50 @@ Read our full [Orb Testing Methodologies]({{site.baseurl}}/2.0/testing-orbs/) do
...
+- Infra Automation
+- Kubernetes
+- Language/Framework
+- Monitoring
+- Notifications
+- Reporting
+- Security
+- Testing
+
+The list of categories can also be obtained by running the `circleci orb list-categories` CLI command. You can view the detailed docs for it [here](https://circleci-public.github.io/circleci-cli/circleci_orb_list-categories.html).
```
Comment (rosieyohannan) [inline jekyll/_cci2/orb-author.md:308]:
```suggestion
The list of categories can also be obtained by running the `circleci orb list-categories` CLI command. You can view the detailed docs for this command [here](https://circleci-public.github.io/circleci-cli/circleci_orb_list-categories.html).
```

File: jekyll/_cci2/orb-author.md [hunk 3]
```diff
@@ -278,3 +278,50 @@ Read our full [Orb Testing Methodologies]({{site.baseurl}}/2.0/testing-orbs/) do
...
+
+#### Add an orb to a category
+
+![](  {{ site.baseurl }}/assets/img/docs/orb-categories-add-to-category.png)
+
+Add your orb to the chosen category by running `circleci orb add-to-category <namespace>/<orb> "<category-name>"`. You can view the detailed docs for the `circleci orb add-to-category` command [here](https://circleci-public.github.io/circleci-cli/circleci_orb_add-to-category.html).
```
Comment (rosieyohannan) [inline jekyll/_cci2/orb-author.md:314]:
```suggestion
Add your orb to your chosen category by running `circleci orb add-to-category <namespace>/<orb> "<category-name>"`. You can view the detailed docs for this command [here](https://circleci-public.github.io/circleci-cli/circleci_orb_add-to-category.html).
```

File: jekyll/_cci2/orb-author.md [hunk 4]
```diff
@@ -278,3 +278,50 @@ Read our full [Orb Testing Methodologies]({{site.baseurl}}/2.0/testing-orbs/) do
...
+
+
+#### Remove an orb from a category
+
+![](  {{ site.baseurl }}/assets/img/docs/orb-categories-remove-from-category.png)
+
+Remove an orb from a category by running `circleci orb remove-from-category <namespace>/<orb> "<category-name>"`. You can view the detailed docs for the `circleci orb remove-from-category` command [here](https://circleci-public.github.io/circleci-cli/circleci_orb_remove-from-category.html).
```
Comment (rosieyohannan) [inline jekyll/_cci2/orb-author.md:321]:
```suggestion
Remove an orb from a category by running `circleci orb remove-from-category <namespace>/<orb> "<category-name>"`. You can view the detailed docs for this command [here](https://circleci-public.github.io/circleci-cli/circleci_orb_remove-from-category.html).
```

File: jekyll/_cci2/orb-author.md [hunk 5]
```diff
@@ -278,3 +278,50 @@ Read our full [Orb Testing Methodologies]({{site.baseurl}}/2.0/testing-orbs/) do
...
+
+#### Viewing an orb's categorizations
+
+![](  {{ site.baseurl }}/assets/img/docs/orb-categories-orb-info.png)
+
+To see what categories an orb is already categorized under, check the output of `circleci orb info <namespace>/<orb>`. It will list the orb’s categorization(s). You can view the detailed docs for the `circleci orb info` command [here](https://circleci-public.github.io/circleci-cli/circleci_orb_info.html).
```
Comment (rosieyohannan) [inline jekyll/_cci2/orb-author.md:327]:
```suggestion
To see which categorizations have been applied an orb, check the output of `circleci orb info <namespace>/<orb>` for a list. You can view the detailed docs for this command [here](https://circleci-public.github.io/circleci-cli/circleci_orb_info.html).
```

File: jekyll/_cci2/orbs-best-practices.md [hunk 6]
```diff
@@ -25,6 +25,11 @@ An orb "slug" is made up of a _namespace_ and _orb_ name separated by a forward
 | company/orb  | company/company |
 {: class="table table-striped"}
 
+#### Categorize your Orb
+{:.no_toc}
+
+Categorizing your orb allows it to be searchable on the [Orb Registry](https://circleci.com/developer/orbs/) by category. To see how you can categorize your orb via the CircleCI CLI, refer to [Orb Authoring Process]({{site.baseurl}}/2.0/orb-author).
```
Comment (rosieyohannan) [inline jekyll/_cci2/orbs-best-practices.md:31]:
```suggestion
Categorizing your orb allows it to be searchable on the [Orb Registry](https://circleci.com/developer/orbs/) by category. To see how you can categorize your orb using the CircleCI CLI, refer to the [Orb Authoring Process]({{site.baseurl}}/2.0/orb-author) guide.
```

**Comment by rosieyohannan** [comment]
Thanks so much for this @lokst!  And thanks for including images too 🎉 ☺️

**Comment by lokst** [comment]
Thanks for the review @rosieyohannan ! I just incorporated your suggestions 🙂

**Review by rosieyohannan — APPROVED** [review]

**Pull request merged**